MAINTAINERS.md

```diff
@@ -88,41 +93,43 @@ A simple majority vote is required to authorize a SPIRE release at a specific co
 #### Checklist
 This section summarizes the steps necessary to execute a SPIRE release. Unless explicitly stated, the below steps must be executed in order.
 
-The following steps must be completed one week prior to release:
+The following steps must be completed by the primary on-call maintainer one week prior to release:
 * Ensure all changes intended to be included in the release are fully merged.
 * Identify a specific commit as the release candidate.
-* Create a draft pull request against the main branch with the updates to the CHANGELOG following [these guidelines](doc/changelog_guidelines.md). This allows those tracking the project to have early visibility into what will be included in the upcoming release and an opportunity to provide feedback. The release date can be set as "TBD" while it is a draft.
-* Raise an issue "Release SPIRE X.Y.Z", and include the release candidate commit hash. Reference the pull request with the updates to the CHANGELOG.
+* Raise an issue "Release SPIRE X.Y.Z", and include the release candidate commit hash.
+* Create the release branch following the guidelines described in [Version branches](#version-branches).
 * If the current state of the main branch has diverged from the candidate commit due to other changes than the ones from the CHANGELOG:
-  * If there is not a version branch for this release, create a branch following the guidelines described in [Version branches](#version-branches).
-  * Create a GitHub project named `Release vX.X.X` to identify the PRs that will be cherry-picked. The project should have two statuses to track the progress: one to identify the PRs to be cherry-picked and one for those that have been merged in the version branch.
   * Make sure that the [version in the branch](pkg/common/version/version.go) has been bumped to the version that is being released and that the [upgrade integration test is updated](test/integration/suites/upgrade/README.md#maintenance).
-  * Cherry-pick into the version branch the commits for all the changes that must be included in the release.
+  * Cherry-pick into the version branch the commits for all the changes that must be included in the release. Ensure the PRs for these commits all target the release milestone in GitHub.
+
+The following steps must be completed by the primary on-call maintainer at least one day before the release (preferably the Monday of the release week):
+* Create a draft pull request against the main branch with the updates to the CHANGELOG following [these guidelines](doc/changelog_guidelines.md). This allows those tracking the project to have early visibility into what will be included in the upcoming release and an opportunity to provide feedback. The release date can be set as "TBD" while it is a draft.
 
-**If this is a major release**, the following steps must be completed before releasing:
+**If this is a major or minor release**, the following steps must be completed by the secondary on-call maintainer before releasing:
```

Comment:
The delegation of this responsibility to the secondary on-call maintainer is a new proposal in order to share some of the release burden load across maintainers for major/minor releases.

Reply:
I think this is a great responsibility to delegate to the secondary, however I'm not sure that we want to do it for _every_ release. It's a fairly heavy lift to walk through everything. Breaking changes are most important, which is why this was originally limited to majors only.

That said, I think we've been pretty lax on this bit in practice, even for majors only

Reply:
Oh, I was using the semver terminology here - by major I meant x.0.0 and minor I meant x.y.0. Were you thinking of major as x.y.0 and minor as x.y.z?

Reply:
spire-tutorials is using v1.5.0, and there is an open PR to use v1.5.0 in spire-examples: https://github.com/spiffe/spire-examples/pull/52

I think once we've updated the examples to use the latest minor version, usually there shouldn't be too many changes we need to make each minor release other than removing/replacing deprecated configs. spire-tutorials has a CI job that runs all the tutorials for you, so testing is not too bad in that repo. For spire-examples, the examples need to be run manually today, but there are also only a handful of examples, so it usually isn't too much effort.

Any other things you can think of that might be time consuming for updating the examples each minor release? For upgrading from v1.0.z/v1.2.z to v1.5.0 in the PRs I referenced, there wasn't all that much to do, so initially I wasn't all that concerned about this being a lot of unexpected work for the secondary.